apps/studio/src/stories/Page/SitePage.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import SitePage from "~/pages/sites/[siteId]"
+import { AdminLayout } from "~/templates/layouts/AdminLayout"
+
+const meta: Meta<typeof SitePage> = {
+  title: "pages/site/[siteId]",
```

Comment:
question: idk the convention about naming stories but had the impression it was named after the component name (and by extension is a name that design uses). Should we use design's name for the component here?

Reply:
Very apt. Will follow figma names instead

Reply:
fixed in upstream PR #308